### PR TITLE
Splash and home view improvements

### PR DESCRIPTION
**Pull request by Jasonvdb**
## Description

Updates splash so tor connects while animations start. 
Updates home view gradient. 
Remove success feedback on alias update. 
Lib version update.

## Motivation and Context

Instead fo waiting for tor to connect before starting the intro animations, it can now be done in parallel. 

## How Has This Been Tested?

Run through all views on simulators and real devices.

## Types of changes

* [x] UI fix (non-breaking change which fixes a UI issue)
* [ ] Functionality bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [x] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I have tested this on multiple simulator devices with different screensizes.
* [x] I'm merging against the `development` branch.
* [x] Commits have been squashed into one commit with a descriptive commit message
* [x] I ran all tests before pushing.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added/changed tests to cover my changes if not UI changes.

### CODE REVIEW

File: MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift [hunk 1]
```diff
@@ -48,7 +48,7 @@ class NodeSyncOperation: Operation {
             return
         }
 
-        NotificationManager.shared.scheduleNotification(title: "Background task", body: "Starting Tor")
+        //NotificationManager.shared.scheduleNotification(title: "Background task", body: "Starting Tor")
```
Comment (lupugabriel) [inline MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift]: Maybe remove this from code? or you plan to add it later?

File: MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift [hunk 2]
```diff
@@ -80,7 +80,7 @@ class NodeSyncOperation: Operation {
 
             self.onComplete(false)
 
-            NotificationManager.shared.scheduleNotification(title: "Background task", body: "Tor connection failed")
+            //NotificationManager.shared.scheduleNotification(title: "Background task", body: "Tor connection failed")
```
Comment (lupugabriel) [inline MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift]: same here

File: MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift (hunk 1 above)
Comment (Jasonvdb) [reply MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift]: Need to use it later to debug background operations

File: MobileWallet/Screens/AppEntry/Splash/SplashViewController.swift [hunk 3]
```diff
@@ -108,7 +111,7 @@ class SplashViewController: UIViewController {
                     attributes.shadow = .active(with: .init(color: EKColor(Theme.shared.colors.feedbackPopupBackground!), opacity: 0.35, radius: 10, offset: .zero))
                     attributes.displayDuration = .infinity
                     attributes.screenInteraction = .forward
-                    SwiftEntryKit.display(entry: self.progressFeedbackView, using: attributes)
+                    //SwiftEntryKit.display(entry: self.progressFeedbackView, using: attributes)
```
Comment (lupugabriel) [inline MobileWallet/Screens/AppEntry/Splash/SplashViewController.swift]: same here

**Pull request merged**